docs/source/onnxruntime/usage_guides/pipelines.mdx

```diff
@@ -35,7 +23,7 @@ Currently supported tasks are:
 * `translation`
 * `image-classification`
 
-## Optimum pipeline usage
+### Optimum pipeline usage
```

Comment:
Do you prefer the rendering of ### rather than ##? Or is there another reason to jump ##?

Reply:
Good catch! Fixed in [7d97309](https://github.com/huggingface/optimum/pull/395/commits/7d97309cf66093866e89178c282ebc022baaab46)